### PR TITLE
UX: list group label indicator should match existing design

### PR DESCRIPTION
**Pull request by MeghnaAJ**
Before:

Screenshot 2023-06-08 at 3 32 30 PM

After:

Screenshot 2023-06-08 at 3 32 17 PM

### CODE REVIEW

**Comment by MeghnaAJ** [comment]
Test failure looks unrelated.

File: app/assets/stylesheets/common/base/_topic-list.scss [hunk 1]
```diff
@@ -264,16 +264,24 @@
...
-        padding: 0 0.3em 0.07em;
-        border-radius: 0.6em;
-        margin-right: 0.25em;
+        display: flex;
+        align-items: center;
+        height: 1.25em;
+        .mobile-view & {
+          height: 1.1em;
+        }
+        color: var(--primary-high);
```
Comment (jjaffeux) [inline app/assets/stylesheets/common/base/_topic-list.scss:280]: it would be nicer to group this color line with the others before mobile view

Comment (MeghnaAJ) [reply app/assets/stylesheets/common/base/_topic-list.scss]: Sure, done. 👍

**Review by oblakeerickson — APPROVED** [review]

**Review by heymarkreeves — APPROVED** [review]

**Comment by MeghnaAJ** [comment]
Thanks @jjaffeux, @oblakeerickson, and @heymarkreeves for the review!

**Pull request merged**